Assets/_BForBoss/_Analytics/Scripts/BForBossAnalytics.cs

```diff
@@ -32,22 +35,28 @@ namespace Perigon.Analytics
             public const string Inverted = "inverted";
         }
     }
+    
+    public enum WorldNameAnalyticsName
+    {
+        MS1RaceCourse,
+        MS2RaceCourse,
+        WeaponsRaceCourse,
+        Unknown
+    }
     #endregion
```

Comment:
I don't think this is a sustainable or scalable solution, to be honest. Cause now everytime a new scenes is made or renamed, this Enum has to be updated as well. Also making the switch statement way too long in the future.

A potential idea is to just to get the scene name for when the name is needed. 
i.e. `SceneManager.GetActiveScene().name` (something like that) and use that instead. And go ahead and change all current scene names to match the convention you think works best. And Ensure that naming standard is used going forward with any new scenes we will create

@unoctium1  what do you think about this as well?

Reply:
I changed it to use `gameObject.scene.name` instead. It's much more isolated and cleaner that way.

Reply:
Sounds good